hieroweb/web/src/hiero/web/RpcObjectManager.java

```diff
@@ -0,0 +1,70 @@
+package hiero.web;
+
+import com.google.gson.Gson;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+import java.util.HashMap;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+// Singleton pattern.
```

Comment:
The manager should be at the very bottom.
But the manager has to be initialized statically to hold the initial object. Not sure how to enforce this constraint in a different way.
Not familiar with dependency injection.